spec/mailboxes/ticket_mailbox_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe TicketMailbox, type: :mailbox do
+  subject { process(mail) }
+
+  let(:body) { Faker::Lorem.paragraphs(number: 2).join(',') }
+  let(:mail_subject) { Faker::Internet.name }
+
+  context 'with valid params' do
+    let(:user) { create(:user) }
+    let(:mail) do
+      Mail.new(from: user.email,
+               to: 'support@rootstrap.com',
+               subject: mail_subject,
+               body:)
+    end
+
+    it 'marks email as delivered when the from receipt is monterrey' do
+      expect(subject).to have_been_delivered
+    end
+
+    it 'creates a new Ticket' do
+      expect { subject }.to change(Ticket, :count).from(0).to(1)
```

Comment:
`.by(1)` ?

Reply:
with zero to 1, we validate that we don't have any tickets in the database. And yes, we can use by(1)

Reply:
@claudiokerekes I think that if the application has been properly configured to clean the database after each case run, then it shouldn't be an issue and it shouldn't make choose `.from(0).to(1)` over `by(1)`